From b1d57df745645f1b1617c48e876c8a3e89a0b564 Mon Sep 17 00:00:00 2001
From: "tao.zeng" <tao.zeng@amlogic.com>
Date: Mon, 23 Jun 2014 11:19:27 +0800
Subject: [PATCH 4376/5965] PD #83628: driver update for AML1218, version 0.80

1. set DCDC1 ~ 3 to Fixed PWM mode first;
2. cacel operation for PWM/PFM switch of dcdc2 during suspend;
3. change register setting for 0x37/0x38/0x39/0x41/0x42/0x4b according david wang.
---
 drivers/amlogic/power/aml_pmu/aml1218-sply.c |  1 +
 drivers/amlogic/power/aml_pmu_of_common.c    | 11 -----------
 2 files changed, 1 insertion(+), 11 deletions(-)

diff --git a/drivers/amlogic/power/aml_pmu/aml1218-sply.c b/drivers/amlogic/power/aml_pmu/aml1218-sply.c
index 70acdb24c975..7f6ab9036a46 100755
--- a/drivers/amlogic/power/aml_pmu/aml1218-sply.c
+++ b/drivers/amlogic/power/aml_pmu/aml1218-sply.c
@@ -1499,6 +1499,7 @@ static int aml1218_battery_probe(struct platform_device *pdev)
     int      ret;
     uint32_t tmp2;
 
+	AML1218_DBG("---> PMU driver version:v0.80\n");
 	AML1218_DBG("call %s in", __func__);
     g_aml1218_init = pdev->dev.platform_data;
     if (g_aml1218_init == NULL) {
diff --git a/drivers/amlogic/power/aml_pmu_of_common.c b/drivers/amlogic/power/aml_pmu_of_common.c
index c0a082e907f0..256252c4249f 100755
--- a/drivers/amlogic/power/aml_pmu_of_common.c
+++ b/drivers/amlogic/power/aml_pmu_of_common.c
@@ -312,17 +312,6 @@ static int aml_pmus_probe(struct platform_device *pdev)
     int    bus_type = -1;
     const  char *str;
 
-    prop = of_find_property(pmu_node, "driver_version", &val);
-    if (!prop) {
-        printk("%s, PMU drvier version not found\n", __func__);
-    } else {
-        if (of_property_read_string(pmu_node, "driver_version", &str)) {
-            printk("%s, PMU drvier version get failed\n", __func__); 
-        } else {
-            printk("----> PMU driver version:%s\n", str); 
-        }
-    }
-
     for_each_child_of_node(pmu_node, child) {
         /* register exist pmu */
         printk("%s, child name:%s\n", __func__, child->name);
-- 
2.19.0

